lazycatlabs/flutter_auth_app
# fix: resolve native splash screen android ios

## What it does

- update launcher icon
- update the library version
- update syntax to generate launcher icon in readme
- resolve the native splash-screen issue on iOS
- resolve the native splash-screen issue on Android 
- resolve the test issue regarding the new Flutter version

## How to test

1. Install the app
2. Observe the view

## Screenshot

### iOS Before
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/5651c960-5c96-43e8-9862-9e2ea4d15673

### iOS After
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/6a170db6-1391-4722-8027-1c70d4a68971

### <12 Android Before
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/4a29a48f-dda8-4510-8cf6-1fd9478fe9c9

### <12 Android After
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/ed8be1bd-c349-460b-9273-051fc04c087e

### >=12 Android Before
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/4657503c-f51d-428c-8625-cce6dd8ed932

### >=12 Android After
https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/57fde055-fa9f-437f-a156-ecdd4aabed72
